ConwaysGameOfLife/Messages.cs

```diff
@@ -10,8 +10,8 @@ public class Messages
 
         public const string QuitGame = "Enter 'q' to quit Game.";
 
-        public const string Run = "Enter 'p' to play game.";
+        public const string Play = "enter 'p' to play game.";

```

Comment:
Why lowercase `enter`?

Reply:
Logic (in code) for PopulateUniverseWithLiveCells:
            // _output.Write(Messages.RequestLiveCell);
            // _output.WriteLine($"or {Messages.Play});
            // loop
            // var locationInput = _input.ReadLine();
            // var location = SetLiveCellLocation(locationInput, universe.GridWidth, universe.GridLength);
            // universe.SwitchCellState(universe.GetCellAtLocation(location.X, location.Y));
            // DisplayUniverse();
Because it is the second part of a 'sentence'